pkg/smokescreen/config.go

```diff
@@ -47,12 +47,11 @@ type Config struct {
 	AllowMissingRole             bool
 	StatsSocketDir               string
 	StatsSocketFileMode          os.FileMode
-	StatsServer                  *StatsServer    // StatsServer
-	ConnTracker                  *sync.Map       // The zero Map is empty and ready to use
-	IdleThresholdSec             time.Duration   // Consider a connection idle if it has been inactive (no bytes transferred) for this many seconds.
-	WgCxns                       *sync.WaitGroup // This wait group tracks *open* (active & idle) connections for graceful shutdown.
-	Healthcheck                  http.Handler    // User defined http.Handler for optional requests to a /healthcheck endpoint
-	IsShuttingDown               atomic.Value    // Stores a boolean value indicating whether the proxy is actively shutting down
+	StatsServer                  *StatsServer // StatsServer
+	ConnTracker                  *conntrack.Tracker
+	IdleThresholdSec             time.Duration // Consider a connection idle if it has been inactive (no bytes transferred) for this many seconds.
```

Comment:
No action required: I am mildly bothered by the `Sec` suffix when this is a `time.Duration` 😛

Reply:
Agreed! time.Duration shouldn't be bound to seconds. Fixed.